core/tauri-codegen/src/embedded_assets.rs

```diff
@@ -196,7 +196,7 @@ impl EmbeddedAssets {
           "import \\*",
           "import (\"|');?$",
           "import\\(",
-          "import (.|\n)+ from (\"|')([A-Za-z\\-]+)(\"|')",
+          "import (.|\n)+ from (\"|')([A-Za-z\\/\.@-]+)(\"|')",
```

Comment:
`"import (.|\n)+ from (\"|')([A-Za-z/\\.@-]+)(\"|')",` ?

Reply:
well, that was weird. Sorry for that 😅